.github/workflows/ci.yml

```
@@ -70,13 +76,19 @@ jobs:
   #   needs: [changes, style, unit-tests]
   #   uses: ./.github/workflows/coverage.yml
 
+  container:
+    if: ${{ needs.changes.outputs.container == 'true' }}
+    needs: [changes, style, security]
+    uses: ./.github/workflows/container.yml
+
```

```suggestion
  permissions:
    contents: read
    packages: write

```

This should fix the CI